src/bindings/python/pyproject.toml

```diff
@@ -1,3 +1,104 @@
+[project]
+version = "2023.3"  # TODO: update automatically?
+requires-python = ">=3.8"  # , <3.12"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11"
+]
+# TODO:
+# description = ""
+# readme = "README.md"
+# authors = [
+#     {name = "OpenVINO", email = "openvino_pushbot@intel.com"},
+# ]
+# license = {text = ""}
+# classifiers = [
+#     "License :: ",
+#     ...
+# ]
+dependencies = [
+    "numpy >= 1.22.4",
+    "openvino-telemetry >= 2023.0",
+]
+
+# [tool.pdm.dev-dependencies]  # will either need --dev or --prod usage
+[project.optional-dependencies]
+linters = [
```

Comment:
How will we keep track of linter packages versions? Or do we agree to just have no version bounds?

Reply:
Currently there are no bounds I am aware of. We might restrict them from below but I would keep them up-to-date all the time. As new standards are constantly added and only new linters are able to check them.